content/ri/using-redisinsight/api.md

```diff
@@ -0,0 +1,178 @@
+---
+Title: RedisInsight REST API
+date: 2020-02-02 00:00:00 +0000
+weight: 1000
+categories: ["RI"]
+path: api/
+nextStep:
+    Title: Memory Analysis
+    href: /docs/features/memory-analysis/
+---
+
+## Add Database API
+
+Used to collect a Token for a registered User.
+
+**URL** : `/api/instance/`
+
+**Method** : `POST`
+
+**Body Type**: JSON
+
+**Auth required** : NO
+
+### Parameters
+
+These are the required parameters for any type of database.
+
+| Parameter     | Type   | Description         |
+|---------------|--------|---------------------|
+| name          | string | A nick name for the Redis database. Any string is valid |
+| dbType        | string | One of `"OSS_NON_CLUSTER"`, `"OSS_CLUSTER"` or `"OSS_SENTINEL"`. For any Redis Enterprise database, use `OSS_NON_CLUSTER` |
```

Comment:
@nishantgeorge Why don't we have a redis enterprise type?  Btw we can have redis enterprise cluster as well.  Why wouldn't we add this so that special features can be detected.  We should even have them for software and cloud no?

Reply:
As we discussed, I've renamed the `dbType` parameter to `connectionType` and also renamed the values:
- `OSS_CLUSTER` => `STANDAONE`
- `OSS_CLUSTER` => `CLUSTER`
- `OSS_SENTINEL` => `SENTINEL`
For enabling vendor and deployment specific features, additional optional parameters may be added in the future.